libs/sdk-core/src/lib/db/search/agentic.ts

```diff
@@ -435,5 +353,118 @@ export namespace Agentic {
         }),
       );
     }
+
+    /*
```

Comment:
any reason why you used simple comment blocks instead of documentation mark `/**` (usually using `/**` your IDE is pre-filling the doc for you?

Reply:
no special reason, the inspiration of the moment I guess